src/abstractions/software/assembler/mockData/index.js

```diff
@@ -1,3 +1,4 @@
 import symbolLess from './symbolLess'
+import symbol from './symbol'
```

Comment:
I think `symbol` is confusing, let's just have `assembly` and `symbolLessAssembly`
Also for this change, update references in the test file
```suggestion
import assembly from './assembly'
```